.github/workflows/cicd.yaml

```diff
@@ -129,6 +129,37 @@ jobs:
           name: safety-report
           path: safety-report.txt
 
+  trivy-misconfiguration:
+    name: trivy config scan
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Render Helm charts
+        run: |
+          mkdir deployment
+          helm template helm > deployment/deployment.yaml
+      - name: Run Trivy on deployment.yaml
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'config'
+          scan-ref: 'deployment'
+          hide-progress: false
+          format: 'table'
+          exit-code: '1'
+          ignore-unfixed: true
```

Comment:
Think for config, there shouldn't be unfixed ones and if there were, I'd probably wanna know

Reply:
Not sure whether same for docker, but might well be the same